City-of-Helsinki/hel-fi-drupal-grants
# AU-881: Multiple profile forms

# [AU-881](https://helsinkisolutionoffice.atlassian.net/browse/AU-881)
# [AU-632](https://helsinkisolutionoffice.atlassian.net/browse/AU-632)
# [AU-615](https://helsinkisolutionoffice.atlassian.net/browse/AU-615)


## What was done


* Added different profile forms for registered communities, unregistered communities and private persons.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/AU-872-asiointiroolit`
  * `make fresh`
* Run `make drush-cr`

## How to test


* Login via Tunnistamo
* When selecting the roles, test all three types of roles and associated forms
* Check validation messages, that all data is saved and retrieved correctly etc



[AU-881]: https://helsinkisolutionoffice.atlassian.net/browse/AU-881?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AU-632]: https://helsinkisolutionoffice.atlassian.net/browse/AU-632?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AU-615]: https://helsinkisolutionoffice.atlassian.net/browse/AU-615?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
